closure/goog/events/browserevent.js

```diff
@@ -223,8 +223,19 @@ goog.events.BrowserEvent.IEButtonMap = [
 goog.events.BrowserEvent.prototype.init = function(e, opt_currentTarget) {
   var type = this.type = e.type;
 
+  /** @type {Touch} */
+  var relevantTouch = null;
+  if (type == goog.events.EventType.TOUCHSTART ||
+      type == goog.events.EventType.TOUCHMOVE) {
+    relevantTouch = e.targetTouches[0];
```

Comment:
After more thinking, and looking at #337, I think it's a mistake to rely on `targetTouches`. With multiple touches on the touch surface `targetTouches[0]` is unlikely to be the "relevant touch". See my comments in #337 for more details, and see the `targetTouches` definition on [mdn](https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent/targetTouches). So as done in #337 I'm tempted to rewrite this whole block as follows:

```js
  /**
   * On touch devices use the first "changed touch" as the relevant touch.
   * @type {Touch}
   */
  var relevantTouch = e.changedTouches ? e.changedTouches[0] : null;
```